contracts/misc/KyberInternalGovernance.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.7.6;
+
+
+import {SafeERC20} from '@openzeppelin/contracts/token/ERC20/SafeERC20.sol';
+import {PermissionAdmin} from '@kyber.network/utils-sc/contracts/PermissionAdmin.sol';
+import {PermissionOperators} from '@kyber.network/utils-sc/contracts/PermissionOperators.sol';
+import {Utils} from '@kyber.network/utils-sc/contracts/Utils.sol';
+import {IERC20Ext} from '@kyber.network/utils-sc/contracts/IERC20Ext.sol';
+import {IKyberGovernance} from '../interfaces/governance/IKyberGovernance.sol';
+import {IRewardsDistributor} from '../interfaces/rewardDistribution/IRewardsDistributor.sol';
+
+
+/**
+    Internal contracts to participate in KyberDAO and claim rewards for Kyber
```

Comment:
Pls add a comment that ppl should delegate to this contract (no deposit required).
This contract will vote on behalf of them and transfer tokens to the recipient

Reply:
Added